### PR TITLE
Task updates from tooltips

### PR DESCRIPTION
**Pull request by miker**
The Tasks section tooltips were merged.
This is the corresponding user guide updates.

### CODE REVIEW

File: userguide/tasks.rst [hunk 1]
```diff
@@ -282,48 +284,47 @@ task.
...
+   | Remote SSH Port                  | integer                     | Only available in  *Rsync over SSH mode*. Allows specifying an SSH port                   |
    |                                  |                             | other than the default of *22*.                                                           |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-   | Rsync mode                       | drop-down menu              | Choices are *Rsync module* or                                                             |
-   |                                  |                             | *Rsync over SSH*.                                                                         |
+   | Rsync mode                       | drop-down menu              | The choices are *Rsync Module mode* or *Rsync over SSH mode*                              |
```
Comment (wblock) [inline userguide/tasks.rst]: As above, "mode" should not be included in the italics.

File: userguide/tasks.rst [hunk 2]
```diff
@@ -282,48 +284,47 @@ task.
...
+   | Short Description                | string                      | Enter a description of the rsync task.                                                    |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
    | Schedule the Rsync Task          | drop-down menu              | Choose how often to run the task. Choices are *Hourly*, *Daily*, *Weekly*, *Monthly*, or  |
-   |                                  |                             | *Custom*. Select *Custom* to open a visual scheduler for selecting minutes, hours, days,  |
-   |                                  |                             | month, and days of week.                                                                  |
+   |                                  |                             | *Custom*. Select *Custom* to open the advanced scheduler                                  |
```
Comment (wblock) [inline userguide/tasks.rst]: End last sentence with a period.

File: userguide/tasks.rst [hunk 3]
```diff
@@ -1745,14 +1750,14 @@ shows the configuration options for Cloud Syncs.
...
    | Direction           | drop-down menu      | *Push* sends data to cloud storage. *Pull* receives data from cloud storage.                            |
    |                     |                     |                                                                                                         |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
-   | Credential          | drop-down menu      | Choose the cloud storage provider credentials from the list of entered :ref:`Cloud Credentials`. The    |
-   |                     |                     | UI automatically tests the credential and displays an error if a connection cannot be made. The         |
+   | Credential          | drop-down menu      | Select the cloud storage provider credentials from the list of available :ref:`Cloud Credentials`.      |
+   |                     |                     | The UI automatically tests the credential and displays an error if a connection cannot be made.         |
```
Comment (wblock) [inline userguide/tasks.rst]: "The credential is tested and an error is displayed if a connection cannot be made."

File: userguide/tasks.rst [hunk 4]
```diff
@@ -282,48 +284,47 @@ task.
...
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
    | Remote Host                      | string                      | Enter the IP address or hostname of the remote system that will store the copy. Use the   |
    |                                  |                             | format *username@remote_host* if the username differs on the remote host.                 |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-   | Remote SSH Port                  | integer                     | Only available in  *Rsync over SSH* mode. Allows specifying an SSH port                   |
+   | Remote SSH Port                  | integer                     | Only available in  *Rsync over SSH mode*. Allows specifying an SSH port                   |
```
[the remark this answers is not in the document]
Comment (miker) [reply userguide/tasks.rst]: done

File: userguide/tasks.rst (hunk 1 above)
Comment (miker) [reply userguide/tasks.rst]: done

File: userguide/tasks.rst (hunk 2 above)
Comment (miker) [reply userguide/tasks.rst]: done

File: userguide/tasks.rst (hunk 3 above)
Comment (miker) [reply userguide/tasks.rst]: done

File: userguide/tasks.rst [hunk 5]
```diff
@@ -1757,8 +1757,8 @@ shows the configuration options for Cloud Syncs.
    |                     |                     |                                                                                                         |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
    | Credential          | drop-down menu      | Select the cloud storage provider credentials from the list of available :ref:`Cloud Credentials`.      |
-   |                     |                     | The UI automatically tests the credential and displays an error if a connection cannot be made.         |
-   |                     |                     | :guilabel:`Save` is disabled until a valid Credential is entered.                                       |
+   |                     |                     | The credential is tested and an error is displayed if a connection cannot be made. :guilabel:`Save` is  |
+   |                     |                     | disabled until a valid Credential is entered.                                                           |
```
Comment (wblock) [inline userguide/tasks.rst]: The last "Credential" should be lower case.

Comment (miker) [reply userguide/tasks.rst]: done

**Pull request merged**